randovania/games/prime1/json_data/Phendrana Drifts.txt

```diff
@@ -2397,6 +2426,19 @@ Extra - aabb: [329.07877, -1248.0734, -17.693329, 398.72052, -1143.9733, 42.7440
                       Slope Jump (Advanced)
                   Scan Visor and Combat/Scan Dash (Advanced)
           Shoot Plasma Beam and Use Grapple Beam
+          All of the following:
+              After Gravity Chamber Item (Lower)
```

Comment:
`and not door lock rando and can shoot wave beam` 🧠

Reply:
Are you referring to dashing off the drone in Chamber Access? Because there is no known way to reach the standable and retain the lock on the entire time (although it was recently discovered that you can dash to the standable instead of bombing to it which I'll add along with the requested changes).

Reply:
No, I'm talking about how you need to reload the room to get the bombus to spawn :)

Reply:
Wave beam can be up there

Reply:
Ah, true. I'll address that then.